Upstream: yes

From 4c7a583e8431964def482e5471f02e6de8e62a7b Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Mon, 27 Jun 2016 12:19:08 +0900
Subject: [PATCH] Fix build with GCC version 6

---
 src/ApvlvParams.cc | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/src/ApvlvParams.cc b/src/ApvlvParams.cc
index b200a5a..f2109ca 100644
--- a/src/ApvlvParams.cc
+++ b/src/ApvlvParams.cc
@@ -36,7 +36,11 @@
 
 namespace apvlv
 {
+#if __cplusplus > 199711L
+  ApvlvParams *gParams = nullptr;
+#else
   ApvlvParams *gParams = NULL;
+#endif
 
   ApvlvParams::ApvlvParams ()
   {
@@ -74,7 +78,11 @@ namespace apvlv
 
   bool ApvlvParams::loadfile (const char *filename)
   {
+#if __cplusplus > 199711L
+    if (filename == nullptr
+#else
     if (filename == NULL
+#endif
 	|| g_file_test (filename, G_FILE_TEST_IS_REGULAR) == FALSE)
       {
 	return false;
@@ -197,7 +205,11 @@ namespace apvlv
       {
 	return it->second.c_str ();
       }
+#if __cplusplus > 199711L
+    return nullptr;
+#else
     return NULL;
+#endif
   }
 
   int ApvlvParams::valuei (const char *s)
-- 
2.20.1

