Upstream: yes

From d432635b9c5ea6c052a2ae1fb71aedec5c4ad57a Mon Sep 17 00:00:00 2001
From: Yaakov Selkowitz <yselkowi@redhat.com>
Date: Fri, 18 Mar 2016 18:40:07 +0100
Subject: [PATCH] Fix gcc6 compile error

References: https://bugzilla.redhat.com/show_bug.cgi?id=1307321
---
 src/ApvlvParams.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ApvlvParams.cc b/src/ApvlvParams.cc
index 15b491c..b200a5a 100644
--- a/src/ApvlvParams.cc
+++ b/src/ApvlvParams.cc
@@ -90,7 +90,7 @@ namespace apvlv
 	return false;
       }
 
-    while ((getline (os, str)) != NULL)
+    while ((getline (os, str)))
       {
 	string argu, data, crap;
 	stringstream is (str);
-- 
2.20.1

